recipes/giflib/config.yml

```diff
@@ -0,0 +1,3 @@
+versions:
```

Comment:
Remove this file, it's not required since you are not using a generic folder for N versions.